File: Java/381.java
public class Solution {
	public static long getSum(int i) {
		long t = 5;
		for (int j = 0; j < 8; j++) {
			t = i*j + 5;
			if (t % 8 == 0)
				return (t/8 - 1);
		}
		return 0;
	}
	public static void main(String[] args) {
		int n = 100000000;
		long result = 0;
		int[] isPrime = new int[n];
		for (int j = 2; j*j < n; j++) {
			if (isPrime[j] == 0) {
				for (int k = j; k*j < n; k++)
					isPrime[j*k] = 1;
			}
		}
		
		
		for (int i = 5; i < n; i++) {
			if (isPrime[i] == 0) {
				result += getSum(i);
			}
		}
		System.out.println(result);
	}
}

File: Python/381.py
import math
def is_prime(x):
	if(x <= 1):
		return False
	if(x == 2):
		return True
	for i in xrange(2,int(math.floor(math.sqrt(x)))+1):
		if(x % i == 0):
			return False
	return True

result = 0

for i in xrange(5,100):
	if(is_prime(i)):
		result += i - 101%i
		print (i - 101%i)

print result